docs/content/getting-started/setup-openfga.mdx

```diff
@@ -53,7 +53,31 @@ All possible configurations and their default values are defined in [config-sche
 You can choose between two storage options:
 
 - In-memory storage. This option is not persistent, i.e. if you restart the server, the data will be lost.
-- Postgres database. This option requires either a local Postgres installation, or a Docker container that runs Postgres. For an example of a Docker setup that includes a Postgres database, see [OpenFGA's Docker compose file](https://github.com/openfga/openfga/blob/main/docker-compose.yaml).
+- Postgres database. This option requires either a local Postgres installation, or a Docker container that runs Postgres.
```

Comment:
```suggestion
- Postgres database. This option requires a running Postgres installation that your OpenFGA server can reach.
```

Whether it is running locally, on docker or on an external network should not be an issue for us.

Reply:
Ah, good point. Thanks!